frontend/src/components/OTPVerification.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { Modal, Button, InputGroup, FormControl } from 'react-bootstrap';
+import styles from './OTPVerification.module.css';
+interface OTPVerificationProps {
+    show: Boolean;
+    onHide: Function;
+}
+export default ({show,onHide}:OTPVerificationProps) => {
+    return (
+        <Modal
+        show={show}
+        onHide={onHide}
+        size="lg"
+        aria-labelledby="contained-modal-title-vcenter"
+        >
+        <Modal.Header closeButton className={styles.OTPverification_modal_body}>
+            <Modal.Title id="contained-modal-title-vcenter" className={"text-align-center"}>
+                Verification Code
+            </Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+            <div className={styles.OTPverification_modal_body}>
+            <div>
+                <h5>Please type verification code sent to</h5>
+                <p>123456789</p>
+            </div>
+            <div>
+                <InputGroup className="mb-3 d-flex justify-content-center ">
+                    <div className="d-flex align-items-center">
+                    <FormControl
```

Comment:
I'm struggling to find the design in Figma (it's a bit of a mess), but I'd recommend against using four separate controls here. You'd either have to manage keyboard intelligently to switch between them as user types text, or you'll end up with a confusing experience.

Can you actually show me a screenshot of how it looks like? And where exactly the design lives on Figma?